File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.java
class Solution {
    public int removeDuplicates(int[] a) {
        int n = a.length;
        int i=0;
        for(int j=0;j<n-1;j++){
            if(a[j]!=a[j+1]){
                a[i++]=a[j];
            }
        }
        a[i++]=a[n-1];
    
        return i;
    }
}

File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var removeDuplicates = function(nums) {
    let unique = 0;
    let left=0,right=1;
    for(right=1;right<=nums.length;right++){
        if(nums[left]!=nums[right]){
            nums[++left]=nums[right]
        }
    }
    return left;
};